VarduhiGhukasyan/chapter_7_30

```diff
@@ -0,0 +1,42 @@
+#include <iostream>
+#include <vector>
+#include <math.h>
+
+using namespace std;
+
+double Evklides(double n,double m) 
+{
+	while (n!=m)
+	{
+		if (n > m) {
+			n -= m;
+		}
+		else m -= n;
+	}
+	return n;
+	
+}
+
+double _sum(const vector<double>& vec, double(*foo)(double,double))
+{
+	double sum = 0;
+
+	for (int i = 0; i < vec.size(); ++i) {
+		for (int j = i + 1; j < vec.size(); ++j) {
+			for (int k = i; k <= j; ++k) {
```

Comment:
խնդիրը չի ուզում գումարներից մաքսիմումը՞ Դու հաշվել ես այդպիսի գումարները ոչ նրանց մաքսիմումը

Reply:
Agree here with Paruyr, but this is a small issue and Varduhi can easily change this if she wants to.